### ISSUE TITLE
Provide Logstash 5.x Compatibility

### ISSUE THREAD

**Issue by PhaedrusTheGreek** [lukewaite/logstash-input-cloudwatch-logs#11]
Great plugin, thank you for your contribution!     We have noticed, however that there are perhaps some (hopefully trivial) dependency issues:

```
Validating /Users/jay/Downloads/logstash-input-cloudwatch_logs-0.9.1.gem
Installing logstash-input-cloudwatch_logs
Plugin version conflict, aborting
ERROR: Installation Aborted, message: Bundler could not find compatible versions for gem "logstash-core":
  In snapshot (Gemfile.lock):
    logstash-core (= 5.0.0)

  In Gemfile:
    logstash-core-plugin-api (>= 0) java depends on
      logstash-core (= 5.0.0) java

    logstash-input-cloudwatch_logs (= 0.9.1) java depends on
      logstash-core (< 3.0.0, >= 1.4.0) java

    logstash-core (>= 0) java

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```

Please let me know if there's any way that Elastic can assist.

**Comment by lukewaite** [comment]
Hi @PhaedrusTheGreek - thanks for opening the issue - I should have been more on top of getting 5.x tested, but I've not yet upgraded myself so it slipped my attention.

Are the only issues the dependency issues? I've not yet had a chance to test myself, but can tag a release easily enough.

**Comment by PhaedrusTheGreek** [comment]
Note that the event object has changed to require use of Getters and Setters instead of direct access

**Comment by nothau** [comment]
+1 for 5.x support

**Comment by lukewaite** [comment]
Logstash 5.x support is now available under [v0.9.3](https://github.com/lukewaite/logstash-input-cloudwatch-logs/releases/tag/v0.9.3) and has been published to rubygems.org.
